README.md

```diff
@@ -102,3 +102,18 @@ Contributing
 =================
 
 Feel free to fork and send us pull requests. Here are the [guidelines for contribution](https://github.com/Murali-group/GraphSpace/blob/master/CONTRIBUTING.md) in GraphSpace.
+
+
+Maintainers
+=================
+
+The Project is maintained by :-
+
+- **T.M Murali ([@tmmurali](https://github.com/tmmurali))**
+- **Aditya Bharadwaj ([@adbharadwaj](https://github.com/adbharadwaj))**
+
+
+Liscence
```

Comment:
Fix the spelling, please.

Reply:
Really sorry for this. I have corrected.